kibana/Dockerfile

```diff
@@ -1,19 +1,33 @@
-FROM openshift/base-centos7
+FROM openshift/base-centos7 
 
 MAINTAINER Eric Wolinetz <ewolinet@redhat.com>
 
-ENV KIBANA_VER 4.1.1
+EXPOSE 5601
```

Comment:
What's in the docker file is really only used as guidance in forming the pod and service. The pod won't expose anything unless so configured.